modules/rhcos-match-vulnerability.adoc

```diff
@@ -3,9 +3,9 @@
 // * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
 :_mod-docs-content-type: CONCEPT
 [id="rhcos-match-vulnerability_{context}"]
-= Vulnerability matching
+= Vulnerability matching on RHCOS nodes
 
 [role="_abstract"]
-Central services, which include Central and Scanner, perform vulnerability matching. Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components.
+Central services, which include Central and Scanner, perform vulnerability matching. Node scanning is performed using the StackRox Scanner. Scanner V4 is not yet supported for node scanning. The StackRox Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components.
```

Comment:
In 4.6 we added RHCOS node scanning to Scanner V4 (tech preview). I believe that will be mentioned in the 4.6 release notes. Do we need to modify anything here, too?